syntax = "proto3";

package seerep;

message GeometryQuery
{
  // uuid of the geometry to query
  string geometryuuid = 1;
}
